framerail/src/lib/server/deepwell/translate.ts

```diff
@@ -0,0 +1,9 @@
+import { client } from "$lib/server/deepwell/index.ts"
+import type { TranslateKeys } from "$lib/types"
+
+export async function translate(locales: string[], keys: TranslateKeys): Promise<object> {
```

Comment:
Could add return type, `TranslatedKeys`, which I believe is `Record<string, string>` (or `Record<string, Optional<string>>`, do we want to change to returning null on no translation found?)

Reply:
I would prefer returning null, since right now the no translation found error obscures all other strings for the keys that do exist, and on a page with a lot of translated strings it becomes a whole bunch of undefineds.